00_Base/src/util/decorators/ValidatedResponseSchema.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { Constructable } from 'typedi';
+import { ResponseSchema } from '../../openapi-spec-helper';
+import { UseAfter } from 'routing-controllers';
+import { ResponseValidationMiddleware } from '../middleware/ResponseValidationMiddleware';
+
+export const validatedResponseParam = 'validatedResponseParam';
+
+export const ValidatedResponseSchema = (
```

Comment:
I'm struggling to understand what's happening here.

Maybe a doc is in order?

Reply:
added